docs/components/table-view.md

```diff
@@ -0,0 +1,1039 @@
+# Table View
+
+Component for displaying data in table format.
+
+:::tip NOTE
+KTableView does not handle data management capabilities like data fetching, functional pagination, sorting or searching. If you are looking for a component that integrates with the data layer, check out [KTable](/components/table).
+:::
+
+<KTableView
+  :data="basicData"
+  :headers="basicHeaders()"
+/>
```

Comment:
issue: the pagination on this example is not functional. If that's expected, we should remove the pagination (need to check the rest of the examples as well)

Also, several of the pagination counts say `1 to 0 of 0`

Reply:
Added hidePagination everywhere where pagination is not relevant in the example.

Reply:
But if it wasn't functional, should the table know not to show it or something?

Reply:
Eh so it can't be fully "smart" like that but I've pushed a change so that it relies on paginationAttributes a bit more to determine if it needs to show pagination (hide if `props.paginationAttributes.totalCount <= props.data.length`)

Reply:
It now looks like the pagination is now not visible on any table examples on the docs page; I'm not sure that's intended/desirable either?

Reply:
It's hidden on all examples because it's irrelevant everywhere except for `paginationAttributes` prop example (there it is visible & functional)